packages/dart_frog_cli/e2e/test/daemon/dev_server_domain_test.dart

```diff
@@ -116,9 +117,10 @@ void main() {
           params: {
             'workingDirectory': projectDirectory2.path,
             'port': project2ServerPort,
-            'dartVmServicePort': project2ServerPort + 1
+            'dartVmServicePort': project2ServerPort + 1,
           },
         ),
+        timeout: const Duration(seconds: 30),
```

Comment:
Can we handle the timeout on a different Pull Request? Otherwise we should update the name of this Pull Request?